packages/neo-one-node-core/src/StackItems/StackItems.ts

```diff
@@ -187,6 +186,6 @@ export const stackItemToJSON = (item: StackItem, context?: Set<StackItem>): Stac
       return { type: 'Pointer', value: pointer.position };
 
     default:
-      throw new InvalidFormatError(`didn't expect stack item of type: ${type} on return`);
+      return { type: 'Any', value: undefined };
```

Comment:
If the type was `'Any'` we would throw, which was not correct and caused incorrect return values when JSON serializing some StackItems. It should actually return a JSON object with type equal to the type and value empty (ie. undefined). `'Any'` is the only type remaining if it makes it to `default` here, so we put `'Any'`.

Reply:
See here: https://github.com/neo-project/neo/blob/master/src/neo/VM/Helper.cs#L230